src/proto/buffer.pro

```diff
@@ -23,6 +23,9 @@ void free_buf_options(buf_T *buf, int free_p_ff);
 int buflist_getfile(int n, linenr_T lnum, int options, int forceit);
 buf_T *buflist_findname_exp(char_u *fname);
 buf_T *buflist_findname(char_u *ffname);
+#ifdef UNIX
+buf_T *buflist_findname_stat(char_u *ffname, stat_T *stp);
+#endif
```

Comment:
This file is generated by the cproto command. The `#ifdef` and `#endif` lines are not needed:
```suggestion
buf_T *buflist_findname_stat(char_u *ffname, stat_T *stp);
```

However, the definitions of `buflist_findname()` and `buflist_findname_stat()` need to be adjusted to make cproto work. E.g.:
```C
    buf_T *
buflist_findname(char_u *ffname)
{
#ifdef UNIX
    stat_T	st;

    if (mch_stat((char *)ffname, &st) < 0)
	st.st_dev = (dev_T)-1;
    return buflist_findname_stat(ffname, &st);
#else
    return buflist_findname_stat(ffname, NULL);
#endif
}

/*
 * Same as buflist_findname(), but pass the stat structure to avoid getting it
 * twice for the same file.
 * Returns NULL if not found.
 */
    buf_T *
buflist_findname_stat(
    char_u	*ffname,
    stat_T	*stp)
{
    buf_T	*buf;

    // Start at the last buffer, expect to find a match sooner.
    FOR_ALL_BUFS_FROM_LAST(buf)
	if ((buf->b_flags & BF_DUMMY) == 0 && !otherfile_buf(buf, ffname
#ifdef UNIX
		    , stp
#endif
		    ))
	    return buf;
    return NULL;
}
```